src/Exception/CannotResolveConfigurationException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Slim Framework (https://slimframework.com)
+ *
+ * @license https://github.com/slimphp/Slim-Console/blob/0.x/LICENSE.md (MIT License)
+ */
+
+declare(strict_types=1);
+
+namespace Slim\Console\Exception;
+
+use Exception;
+
+class CannotResolveConfigurationException extends Exception
```

Comment:
Shouldn't that be named as `CannotResolveConfigException` instead?

Reply:
I need to get it through my head that we're using `Config` and not `Configuration` 😄